src/diffusers/loaders/peft.py

```diff
@@ -71,30 +74,27 @@ def _maybe_adjust_config(config):
         key_rank = rank_pattern[key]
 
         # try to detect ambiguity
-        # `target_modules` can also be a str, in which case this loop would loop
```

Comment:
Any specific reason why this was removed?

Reply:
Let me add this back.